vra7/resource_vra7_deployment.go

```diff
@@ -97,21 +97,13 @@ func resourceVra7Deployment() *schema.Resource {
 			"deployment_configuration": {
```

Comment:
This was the only file pertaining to business logic that needed to be changed to support 12.